### ISSUE TITLE
Pre-populating repeater fields results in duplicate values

### ISSUE THREAD

**Issue by mikeymeister** [verbb/formie#2193]
### Describe the bug

When trying to prepopulate a repeater field with two rows, both rows have the value of the last entry in the array used to populate it. Strangely, if I deliberately mistype the subfield handle for the last item, both rows in the field are blank, so it seems like its creating rows based off the number of array items, but only using the last one to populate the values, rather than the last one overriding the first.
Thanks.

### Steps to reproduce

1. Prepopulate a repeater fields values in the template
```
{% do craft.formie.populateFormValues( form , {
        'name': 'firstname lastname',
        ...
        'services': [
            {
                 'service': 'Service 1'
            },
             {
                 'service': 'Service 2'
            },   
        ]
}) %}
```
2. Render the form
3. Input has duplicated the last item for both rows.
![Screenshot 2024-12-16 at 20-23-45 ](https://github.com/user-attachments/assets/282cc9fd-3a81-4ef5-89c4-c26d3fe24f37)

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: Yes

### Craft CMS version

5.5.6.1

### Plugin version

3.0.14

### Multi-site?

Yes

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-5 as 3.0.14"`.

**Comment by engram-design** [comment]
Fixed in [3.0.15](https://github.com/verbb/formie/releases/tag/3.0.15)
